float64 timeStamp      # ROS time-stamp
float64 angleMotor     # Motor angle
float64[] angle        # Finger angles
